pwnlib/tubes/ssh.py

```diff
@@ -667,8 +667,12 @@ def process(self, argv=None, executable=None, tty=True, cwd=None, env=None, time
                 See ``stdin``.
             preexec_fn(callable):
                 Function which is executed on the remote side before execve().
+                This **MUST** be a self-contained function -- it must perform
+                all of its own imports, and cannot refer to variables outside
+                its scope.
```

Comment:
You can't, actually.  And if you can, it's more complex than I care to worry about.  If things break, the exception is thrown on the *other* side and all we have is the Python stack trace -- which appears as part of the `tube` data.  There's not a way around this that I'm aware of.